shardingsphere-scaling/shardingsphere-scaling-dialect/shardingsphere-scaling-opengauss/src/main/java/org/apache/shardingsphere/scaling/opengauss/component/OpenGaussWalDumper.java

```diff
@@ -57,6 +61,8 @@
     private final OpenGaussLogicalReplication logicalReplication = new OpenGaussLogicalReplication();
 
     private final WalEventConverter walEventConverter;
+    
+    private String slotName = OpenGaussLogicalReplication.SLOT_NAME_PREFIX;
```

Comment:
no, this slotName is changable in initReplication function. to set default to SLOT_NAME_PREFIX just to avoid NPE issue when getUniqueSlotName throw EXception.

Reply:
So could we set it as local variable in method and initialize it on demand?

Reply:
So could we set it as local variable in method and initialize it on demand?

Reply:
no, this filed init need Connection, but we not save Connection as class field.
it's not worth the cost to init this field by create a connection to database.

Reply:
OK